src/services/directoryServices/__tests__/MediaDirectoryService.spec.js

```diff
@@ -129,46 +129,6 @@ describe("Media Directory Service", () => {
         directoryName: imageDirectoryName,
       })
     })
-    mockGitHubService.getRepoInfo.mockResolvedValueOnce({
```

Comment:
could i get some context behind the mass deletion fo this chunk? is this for media-utils test?

Reply:
Yup this was no longer easy to test in the directory service itself - the equivalent tests (for private repos) has been moved into media-utils!